Comparison Engine/Comparison Engine/Form Classes/ButtonListMethods.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Comparison_Engine.Base_Classes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows.Forms;
+
+namespace Comparison_Engine.Form_Classes
+{
+    class ButtonListMethods
+    {
+
+
+        //BAR BUTTON FUNCTIONS
+        public static void CreateBarButton(Bar bar, Form1 mainApplication)
+        {
+            Button button = new Button();
+            ConfigureBarButton(button, bar, mainApplication);
+            BindBar(button, bar, mainApplication);
+        }
+
+        private static void BindBar(Button button, Bar bar, Form1 mainApplication)
+        {
+            button.Click += (sender, EventArgs) => { BarButtonClick(bar, mainApplication); };
+        }
+
+        private static void BarButtonClick(Bar bar, Form1 mainApplication)
+        {
+            ChildFormMethods.OpenChildFormBar(bar, mainApplication);
+            mainApplication.mapController.RemoveOverlays(mainApplication.map);
+            mainApplication.mapController.ShowRoute(mainApplication.map, bar.barLocation);
+        }
+        private static void ConfigureBarButton(Button button, Bar bar, Form1 mainApplication)
+        {
+            mainApplication.panelSubList.Controls.Add(button);
+            button.Text = (bar.barName);
+            button.Dock = DockStyle.Top;
+        }
+
+
+
+        //DRINK BUTTON FUNCTIONS
+        public static void CreateDrinkButton(Drink drink, Form1 mainApplication)
+        {
+            Button button = new Button();
+            ButtonListMethods.ConfigureDrinkButton(button, drink, mainApplication);
```

Comment:
Any specific reason why you use ButtonListMethods inside the class itself?

Reply:
Forgot to remove this when moving code from Form1 to an external class